json_schemas/api_bridge_delete_user.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "required": [
+    "username",
+    "user_to_delete"
+  ],
+  "properties": {
+    "username": {
```

Comment:
Username should be read from the request credentials, you would not want a user to create a request, authenticating as user and put an admins username in here